### DISCUSSION TITLE
Application BaramFlow not working anymore on Windows 10

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#149] [General]
Hello,

I ran a calculation on BaramFlow 24.4 this morning and I had a crash, I had to shut down the computer everything was freeze.

Now I am not able to start the BaramFlow application, nothing happens when I execute the shortcut. I unistall and reinstall Baram but it doesn't work anymore on my computer ! BaramMesh works.

Do you have an idea to help me? 

Best regards

Julien

**Comment by kdg5424** [comment]
Hello Julien. This is Donggyu.

I'm very sorry to hear that. But the clue for the problem is not enough.
I can't be sure why BARAM doesn't work suddenly.
Have you changed your PC setting or installed other programs?
Please check it.

Best Regards

Donggyu Kim

**Comment by torodecamargue** [comment]
I found a .lock file generated after the freeze. I delete it and now it works!

**Comment by kdg5424** [comment]
I'm glad to hear that Julien.

Best Regards

Donggyu Kim
